editor.planx.uk/src/@planx/components/PlanningConstraints/Public.tsx

```diff
@@ -55,23 +55,11 @@ function Component(props: Props) {
   const wktPolygon: string | undefined =
     siteBoundary && stringify(siteBoundary);
 
-  // Configure which planx teams should query Digital Land (or continue to use custom GIS) and set URL params accordingly
-  //   In future, Digital Land will theoretically support any UK address and this list won't be necessary, but data collection still limited to select councils!
-  const digitalLandOrganisations: string[] = [
-    "opensystemslab", // for UK-wide testing, subject to data availability
-    "barnet",
-    "birmingham",
-    "buckinghamshire",
-    "canterbury",
-    "camden",
-    "doncaster",
-    "gloucester",
-    "lambeth",
-    "medway",
-    "newcastle",
-    "southwark",
-    "st-albans",
-  ];
```

Comment:
I'll update `teams.settings.hasPlanningData` for these councils on merge :heavy_check_mark: 

Please update a single record on the pizza to test: 
- You should see the "{Council} is not a supported Local Authority" error before updating `teams.settings`
- You should successfully query constraints as expected after updating

Reply:
I wonder if it's worth capturing this as a Hasura migration? The value of `hasPlanningData` will always be true across all environments, and would mean we wouldn't have to add the column to the sync script.

Reply:
Getting the expected results for `true`, `null` and `false` values 👍